proxy/healthy_endpoints_test.go

```diff
@@ -101,6 +101,17 @@ func setupServices(t *testing.T, healthy, unhealthy int) []string {
 	return services
 }
 
+func formatServicesString(services []string) string {
```

Comment:
I like this function. :-) Really helpful to avoid fmt.Printf arguments mess.

Reply:
Can't you use strings.Join() to achieve the same?

Reply:
I don't know will check, here we need to include '"' in the string I don't think that's feasible from strings.Join but will check